### ISSUE TITLE
Move to a command line arg parser

### ISSUE THREAD

**Issue by swdgit** [ezekg/swa-dashboard#17]
I used to leverage commons-cli for java based cli apps and figure it wouldn't hurt to do the same here. You would get error checking, required vs optional, default values, descriptions etc. This will also reduce the maintenance burden as more options are added.

**Comment by ezekg** [comment]
Agreed. This seems like a good idea. I'm sure there are some simplistic CLI libs for Node.

**Comment by swdgit** [comment]
There are a brazillion of them. My first goal will be to narrow down the list to 2 or maybe 3 potentials and then play with them to see which is fits the bill best. This is where I feel npm needs a vetting process along the lines of Apache Commons. If I'm missing something let me know.

**Comment by ezekg** [comment]
@swdgit [yargs](https://www.npmjs.com/package/yargs) seems like it would be a good choice. [A fork of this repo is using it here](https://github.com/ThatCrazyIrishGuy/ryanair-dashboard/blob/master/index.js?ts=2#L31-L45).

**Comment by swdgit** [comment]
I had looked into command-line-args and command-line-arguments as they are higher up on the popularity and maintenance list for npm. I'll look at yargs and see how that would work.

**Comment by ezekg** [comment]
Whatever works. 👍
